packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart

```diff
@@ -23,21 +23,19 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
   AtOnboardingPreference atOnboardingPreference;
 
   AtOnboardingServiceImpl(atsign, this.atOnboardingPreference) {
-    _atSign = AtUtils.formatAtSign(atsign)!;
+    atsign = AtUtils.formatAtSign(atsign)!;
```

Comment:
you can change this to AtUtils.fixAtSign(AtUtils.formatAtSign(atsign));
you need not change the passed parameter atsign inside the constructor

Reply:
addressed review comments as part of 3ecbf3191d2a90e5b21cdf9aa8b6da7421e78437